artifacts/deploy/karmada-aggregation-apiserver.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: karmada-aggregation-apiserver
+  namespace: karmada-system
+  labels:
+    app: karmada-aggregation-apiserver
+    apiserver: "true"
+spec:
+  selector:
+    matchLabels:
+      app: karmada-aggregation-apiserver
+      apiserver: "true"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: karmada-aggregation-apiserver
+        apiserver: "true"
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - name: karmada-aggregation-apiserver
+          image: swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-aggregation-apiserver:latest
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: k8s-certs
+              mountPath: /etc/kubernetes/pki
+              readOnly: true
+            - name: kubeconfig
+              subPath: kubeconfig
+              mountPath: /etc/kubeconfig
+          command:
+            - /bin/karmada-aggregation-apiserver
+            - --kubeconfig=/etc/kubeconfig
+            - --authentication-kubeconfig=/etc/kubeconfig
+            - --authorization-kubeconfig=/etc/kubeconfig
+            - --etcd-servers=https://etcd-client.karmada-system.svc.cluster.local:2379
+            - --etcd-cafile=/etc/kubernetes/pki/server-ca.crt
+            - --etcd-certfile=/etc/kubernetes/pki/karmada.crt
+            - --etcd-keyfile=/etc/kubernetes/pki/karmada.key
+            - --tls-cert-file=/etc/kubernetes/pki/karmada.crt
+            - --tls-private-key-file=/etc/kubernetes/pki/karmada.key
+            - --audit-log-path=-
+            - --feature-gates=APIPriorityAndFairness=false
+            - --audit-log-maxage=0
+            - --audit-log-maxbackup=0
+          resources:
+            requests:
+              cpu: 100m
+              memory: 20Mi
```

Comment:
20MiB, is it enough?

Reply:
It does seem to be too small. 

It's ok during my local testing.

Let me remove it first just as `karmada-apiserver`.